.github/workflows/tests@v1.yml

```diff
@@ -216,17 +217,17 @@ jobs:
           cp --parents ./**/target/*-reports/*.xml cassandra-${{ matrix.cassandra-version }}/
 
       - name: Upload test results
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
         if: success() || failure()
         with:
-          name: test-results
+          name: test-results-${{ matrix.java-version }}-${{ matrix.cassandra-version }}
           path: "*/**/target/*-reports/*.xml"
 
       - name: Upload CCM logs
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
         if: ${{ failure() }}
         with:
-          name: ccm-logs-cassandra-${{ matrix.cassandra-version }}
+          name: ccm-log-scylla-${{ matrix.java-version }}-${{ matrix.cassandra-version }}
```

Comment:
nit: Should it be ccm-log-scylla or ccm-log-cassandra here?

Reply:
thanks, fixed